hackathon_site/dashboard/frontend/src/components/general/Navbar/Navbar.test.js

```diff
@@ -11,17 +11,17 @@ describe("<Navbar />", () => {
         expect(asFragment()).toMatchSnapshot();
     });
 
-    it("Adds the active class to only the active page", () => {
-        const pagesAndPaths = [
-            ["Dashboard", "/"],
-            ["Orders", "/orders"],
-            ["Teams", "/teams"],
-            ["Reports", "/reports"],
-            ["Inventory", "/inventory"],
-            [`Cart (${cartQuantity})`, "/cart"],
-        ];
+    const pagesAndPaths = [
+        ["Dashboard", "/"],
+        ["Orders", "/orders"],
+        ["Teams", "/teams"],
+        ["Reports", "/reports"],
+        ["Inventory", "/inventory"],
+        [`Cart (${cartQuantity})`, "/cart"],
+    ];
 
-        pagesAndPaths.map(([label, path]) => {
+    pagesAndPaths.map(([label, path]) => {
+        it("Adds the active class to only the active page", () => {
```

Comment:
Can you update this message with a detail about which one it's doing? Otherwise the output will be this same message 6 times. Something like 

```javascript
it(`Adds the active class to ${label} when on ${path}`, ...)
```